ItemLevelDownloader/Providers/fileShareProvider.ts

```diff
@@ -0,0 +1,22 @@
+﻿import * as models from "../Models"
```

Comment:
as long as Bishal is renaming this to local file system provider, we should be fine. let's not end up having file share & local file system provider at the end.